kong/clustering/init.lua

```diff
@@ -108,7 +108,9 @@ function _M:init_cp_worker(plugins_list)
   -- their data planes
   kong.worker_events.register(handle_dao_crud_event, "dao:crud")
 
-  self.json_handler:init_worker(plugins_list)
+  if kong.configuration.legacy_hybrid_compatibility then
```

Comment:
Maybe we could squash this and `legacy_hybrid_protocol` into one setting (`hybrid_protocol = wrpc,legacy|wrpc|legacy`). WDYT? I know we're just trying to get an issue resolved for 3.0 as simply as possible so feel free to disregard.

Reply:
`legacy_hybrid_protocol` is a hidden config, while `legacy_hybrid_compatibility` is exposed to users. However, it doesn't matter now as we are automatically disabling old protocol when no old DP connects.